Documentation/ABI/JSON.md

```diff
@@ -197,10 +197,20 @@ sufficient information to display the event in a human-readable format.
   "runEnded" ; additional event kinds may be added in the future
 
 <issue> ::= {
+<!--
```

Comment:
I haven't bothered to add other underscored stuff prior to review. We can leave it out for now and add it as part of your proposal when you make it.

I _think_ this is going to require a new ABI revision though.